### ISSUE TITLE
/browse path doesn't works

### ISSUE THREAD

**Issue by inthelovedotorg** [rajput-hemant/infinitunes#53]
the path /browse doesn't works and throws an error when clicked on browse button

**Comment by rajput-hemant** [comment]
hey @inthelovedotorg,
my bad, i forgot to optimize the site for the mobile experience, feel free to raise a PR if youre able to
the idea was to shift the 'Discover' sidebar section to the 'Browse' page for smaller screens

**Comment by inthelovedotorg** [comment]
@rajput-hemant yeah the mobile has alot of things that is missing some of them are as follows 

- the player when playing on mobile should have previous, skip, loop shuffle etc, things on it.
- when the song is currently playing and if you add another song to queue while a song is currently playing, the currently playing songs restarts whenever you add a song to queue.
- the songs should also autoplay according to the same genre when a song is played.

**Comment by rajput-hemant** [comment]
hey @inthelovedotor,
i completely agree with you, i had plans to make the player more functional for mobile devices as well, but i havent had the time to work on it yet
if youre interested, id really appreciate your help in addressing these issues!

**Comment by inthelovedotorg** [comment]
> hey @inthelovedotor,
> i completely agree with you, i had plans to make the player more functional for mobile devices as well, but i havent had the time to work on it yet
> if youre interested, id really appreciate your help in addressing these issues!

I sure will try my best
